.github/workflows/test.yaml

```
@@ -0,0 +1,31 @@
+
+      - uses: gradle/actions/setup-gradle@v3
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: Build (skip tests)
+        run: ./gradlew clean bootJar -x test --no-daemon
+
+      # Dockerfile 유효성만 체크(푸시X)
+      - name: Docker build (no push)
+        run: docker build -t temp:test -f deploy/Dockerfile .
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)